Assets/Scripts/Spawn System/PlayerRespawn.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+//attach to player
+public class PlayerRespawn : MonoBehaviour
+{
+    GameObject player;
```

Comment:
Can remove player object (see comment below where the object can be referenced using this)